conda/recipe/build.sh

```diff
@@ -159,7 +159,11 @@ echo "DALI_PATH is ${DALI_PATH}"
 pushd $SRC_DIR/dali_tf_plugin/
 mkdir -p dali_tf_sdist_build
 cd dali_tf_sdist_build
+
+CUDA_VERSION_STR=$(echo $(ls /usr/local/cuda/lib64/libcudart.so*)  | sed 's/.*\.\([0-9]\+\)\.\([0-9]\+\)\.\([0-9]\+\)/\1.\2/')
```

Comment:
Maybe it is better to move it next to https://github.com/NVIDIA/DALI/blob/master/conda/build_conda_packages.sh#L11  (just remember to make that new var visible inside the build.sh - see https://github.com/NVIDIA/DALI/blob/master/conda/recipe/meta.yaml#L50)